doc/manual/parallel-computing.rst

```diff
@@ -451,36 +550,33 @@ preemptively. This means context switches only occur at well-defined
 points: in this case, when :func:`remotecall_fetch` is called.
 
 
-Channels
```

Comment:
@amitmurthy - Can you spell out what is precisely meant by inter-task vs inter-process communication? I think I conflated them in what I wrote because the examples seem to emphasize the similarities. Perhaps we could rewrite this section as:

- `Channels` and `RemoteChannels`
    - Both are `AbstractChannels` and implement `fetch`, `put!`, `take!`, etc.
    - Provide an example of the above commands
    - Provide an example showing how a `Channel` and `RemoteChannel` behave differently.